watcher/lib/influxdb/influxdb.go

```diff
@@ -80,6 +121,39 @@ func (c *Client) Setup() error {
 	return nil
 }
 
+// ManageUsers manages users in the database
```

Comment:
`manages` is not too descriptive: how exactly does it manage or do to users?
`EnsureUsers` or `UpsertUsers` - i.e. create users if not existing would be mildly better